### DISCUSSION TITLE
Toggle functionality when opening page from cefsharp or directly

### DISCUSSION THREAD

**Discussion by shaharzi** [cefsharp/CefSharp#3365] [General]
Hello,

i have a web page that has two options for accessing. one is when it is hosted with cefsharp and the second is accessing directly to the site. i want to add functionality in the web page to know in which option I accessed the page, to be able to remove the cef binding and change function. 

what is the best way to do that?

thanks.

**Comment by amaitland** [comment]
In javascript check if `CefSharp`(or `cefSharp`) is defined in the global scope e.g. `window.cefSharp` or `window.CefSharp` will exist when running in `CefSharp` (if you've renamed them then you can adjust the naming).
